Data-Pro-Test/Views/Recipes/Index.cshtml

```diff
@@ -0,0 +1,40 @@
+﻿
+@{
+    ViewBag.Title = "Index";
+}
+
+<h2>Index</h2>
+
+<div>
+    <img class="header-img" src="~/Content/Images/header.jpg" />
+</div>
+<div class="row">
+    <div class="container content">
+        @RenderPage("~/Views/Recipes/_receipt.cshtml")
+
+        @RenderPage("~/Views/Recipes/_enrolment.cshtml")
+
+        @*<partial name="~/Views/Recipes/_receipt.cshtml" />
+        <partial name="~/Views/Recipes/_enrolment.cshtml" />*@
+    </div>
+</div>
+
+<footer class="nav-footer footer-style">
+    <img class="logo" src="~/Content/Images/logo.png" />
+</footer>
+
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
metadata and links are for the site and not only for the  receipr page . so they neeed to be in the _leyout partial